### ISSUE TITLE
Accordions violate accessibility requirements of WCAG 2.1

### ISSUE THREAD

**Issue by octavpo** [jquery/jquery-ui#2403]
When I run a scan with axe-core DevTools on a page containing an accordion made with your widget, I get a critical error indicating that for the element with a 'tablist' role "Element has children which are not allowed: [role=tabpanel]". The [ARIA Authoring Practices Guide (APG)](https://www.w3.org/WAI/ARIA/apg/) examples at https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/ seems to show accordions shouldn't use tab roles at all.

**Comment by mgol** [comment]
Thanks for the report. Does the issue you describe exist when jQuery UI 1.12.1 is used or only with jQuery UI 1.13.0 or newer?
